### PR TITLE
Problem: Cosmos SDK not upgraded to include patch for dragonberry

### PR DESCRIPTION
**Pull request by tomtau**
Solution: Upgrade cosmos-sdk to 0.46.3 + update submodules

based on #883

### CODE REVIEW

**Comment by tomtau** [comment]
@yihuang @devashishdxt it seemed the upgrade integration test failed due to a different Cosmovisor version, so I reverted nix + git submodule to the original one, but not sure if correct

**Comment by tomtau** [comment]
oh, grpc would fail then

File: x/nft/client/cli/tx.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func GetCmdIssueDenom() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			optionsContent, err := ioutil.ReadFile(schema)
+			optionsContent, err := os.ReadFile(schema)
```
[the remark this answers is not in the document]
Comment (tomtau) [reply x/nft/client/cli/tx.go:76]: https://github.com/crypto-org-chain/chain-main/issues/872

**Review by yihuang — APPROVED** [review]
LGTM, missing a changelog entry?

**Comment by tomtau** [comment]
> LGTM, missing a changelog entry?

I'll aim to release on Monday, so may update ibc-go to the official release (if there's one) by that time

**Comment by devashishdxt** [comment]
New version released: https://github.com/cosmos/ibc-go/releases/tag/v5.0.1

**Pull request merged**